app/code/Magento/InventoryAdminUi/Test/Mftf/Test/LoggedInCustomerOrderedGroupedProductWithChildProductsAssignedToDefaultStockFromHomepageTest.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="LoggedInCustomerOrderedGroupedProductWithChildProductsAssignedToDefaultStockFromHomepageTest"
+          extends="AdminCreateOrderWithGroupedProductDefaultStockTest">
+        <annotations>
+            <stories value="Orders with grouped products"/>
+            <title value="Logged in Customer ordered Grouped product with child products assigned to default Stock from Homepage"/>
+            <testCaseId value="https://app.hiptest.com/projects/69435/tests-plan/folders/419537/scenarios/1437519"/>
+            <severity value="CRITICAL"/>
+            <group value="msi"/>
+            <group value="multi_mode"/>
+        </annotations>
+        <before>
```

Comment:
So much removals may indicate that test should not be extended. It might be better to find appropriate action groups and refactor both tests.